acceptance/api/v1/applications_test.go

```diff
@@ -721,7 +756,7 @@ var _ = Describe("Apps API Application Endpoints", func() {
 					deploy := &models.DeployResponse{}
 					err = json.Unmarshal(bodyBytes, deploy)
 					Expect(err).NotTo(HaveOccurred())
-					Expect(deploy.Route).To(MatchRegexp(appName + `.*\.omg\.howdoi\.website`))
```

Comment:
@richard-cox the domains will become CRUD'able here: https://github.com/epinio/epinio/issues/931 so no need to worry about it yet. By default, if you ignore the domains feature, it will work like before. 

If you want to support the domains feature on the UI you will have to add it to the "create" request ([like we do in this test](https://github.com/epinio/epinio/pull/942/files#diff-135b5db4b4dbd391b4bdfe06ab6c8473420c4c42d6db23e4b67a7f75f5e9ae78R144-R149))

Also the "update" request (when you edit an application) now accepts the "domains" param: https://github.com/epinio/epinio/pull/942/files#diff-98860d4deb0991f533905d10afc8406ee9efd1c9d5b36e1901dde60d216bfd6eR169

I think you have some proxy printing out requests as they fly (when using the cli) so you should be able to discover the json structure, if not let me know and I can send you some examples.